src/app/components/simple-editable-list/SimpleEditableList.jsx

```diff
@@ -59,7 +60,7 @@ export default class SimpleEditableList extends Component {
                                     field={field}
                                     handleEditClick={this.handleEditClick}
                                     handleDeleteClick={this.handleDeleteClick}
-                                    disabled={this.props.disableActions}
+                                    disabled={this.props.disableActions || this.props.disableEditFunctionality}
```

Comment:
I see we always use `disableEditFunctionality` together with `disableActions`, could we reuse the existing prop instead?

Reply:
I've also taken out the `disableEditFunctionality` and update the `disableActions` prop so that the component is disabled if there is any metadata coming through for that particular field.